specs/PushNotifications/ToastNotifications-spec.md

```diff
@@ -0,0 +1,551 @@
+# App Notifications in Windows App SDK
+
+# Background
+
+App Notifications are UI popups that that contain rich text, controls and images to
+display a message to the user. They are developer driven and originate from a target application
+that is installed by a user on the local device. It is not sufficient for a notification popup on the
```

Comment:
Below it says they can originate from the network. I presume you mean they "target" or are "associated with" an app on the device? Also adding "local" seems like too much specificity. Suggest maybe "They are developer-driven and are associated with an installed app."